sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/AmqpSendLink.java

```diff
@@ -45,4 +45,11 @@
      * Gets the context for this AMQP send link.
      */
     ErrorContext getErrorContext();
+
+    /**
+     * The remote endpoint path this link is connected to.
+     *
+     * @return The remote endpoint path this link is connected to.
+     */
+    String getHostname();
```

Comment:
Should this be called the `getEndpoint()` since the javadoc refers this to endpoint?

Reply:
It's that weird string like: eh-namespace.servicebus.windows.net. I'm not sure it's an endpoint because we can't use https:// or sb:// to connect to it. In the track1 code they called it "hostname" so I maintained that name.

Reply:
yeah, the javadoc was the same from above `EntityPath`. Updated it.